en/releases/release-1.0.0-beta.2.md

```diff
@@ -0,0 +1,151 @@
+---
+title: TiDB Operator 1.0 Beta.2 Release Notes
+---
+
+# TiDB Operator 1.0 Beta.2 Release Notes
+
+Release date: May 10, 2019
+
+TiDB Operator version: 1.0.0-beta.2
+
+## v1.0.0-beta.2 What’s New
+
+### Stability has been greatly enhanced
+
+- Refactored e2e test
+- Added stability test, 7x24 running
+
+### Greatly improved ease of use
+
+- One-command deployment for AWS, Aliyun
+- Minikube deployment for testing
+- Tkctl cli tool
+- Refactor backup chart for ease use
+- Refine initializer job
+- Grafana monitor dashboard improved, support multi-version
+- Improved user guide
+- Contributing documentation
+
+### Bug fixes
+
+- Fix PD start script, add join file when startup
+- Fix TiKV failover take too long
+- Fix PD ha when replcias is less than 3
+- Fix a tidb-scheduler acquireLock bug and emit event when scheduled failed
+- Fix scheduler ha bug with defer deleting pods
+- Fix bug when using shareinformer without deepcopy
+
+### Other improvements
+
+- Remove pushgateway from TiKV pod
+- Add GitHub templates for issue reporting and PR
+- Automatically set the scheduler K8s version
+- Swith to go module
+- Support slow log of TiDB
+
+## Detailed Bug Fixes And Changes
+
+- Don't initialize when there is no tidb.password ([#282](https://github.com/pingcap/tidb-operator/pull/282))
+- fix join script ([#285](https://github.com/pingcap/tidb-operator/pull/285))
+- Document tool setup and e2e test detail in Contributing.md ([#288](https://github.com/pingcap/tidb-operator/pull/288))
+- Update setup.md ([#281](https://github.com/pingcap/tidb-operator/pull/281))
+- Support slow log tailing sidcar for tidb instance ([#290](https://github.com/pingcap/tidb-operator/pull/290))
+- Flexible tidb initializer job with secret set outside of helm ([#286](https://github.com/pingcap/tidb-operator/pull/286))
+- Ensure SLOW_LOG_FILE env variable is always set ([#298](https://github.com/pingcap/tidb-operator/pull/298))
+- fix setup document description ([#300](https://github.com/pingcap/tidb-operator/pull/300))
```

Comment:
How about making the capitalization and wording style consistent for this whole section? It might be a lot of work.

Reply:
Done.